tyr/clusters/autoscaling.py

```diff
@@ -90,7 +109,14 @@ def create_autoscaling_group(self):
             self.log.info("Creating new autoscaling group: {g}"
                           .format(g=self.autoscaling_group))
 
+	    # Convert our tags list into something that AWS can understand:
+	    aws_tags = list()
+	    for tag in self.tags:
+		self.log.info("Adding tag [" + str(tag['name']) + "] with value [" + str(tag['value']) + "]")
+		aws_tags.append(Tag(resource_id=self.autoscaling_group, key=tag['name'], value=tag['value'], propagate_at_launch=True))
+	    
```

Comment:
Not just an issue on this line, but I've noticed a couple places in this PR where tabs are being used.

FWIW, spaces are preferred in Python and, while some people use tabs, let's not mix the two indentation styles.